data-model.sql

```diff
@@ -471,7 +471,7 @@ ALTER TABLE query_organization OWNER to pulse;
 
 CREATE TABLE patient_record (
 	id bigserial not null,
-	dob date,
+	dob varchar(100),
 	ssn varchar(15),
```

Comment:
This SSN is a different length than the one for Patient; I think they should both be the same (and likely 15)